.gitignore

```diff
@@ -7,3 +7,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.vscode
```

Comment:
This shouldn't be here, but rather in your global .gitignore file, see https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
I could have sworn I just fixed this! How have the tabs gotten larger. sigh.